frontend/src/APIClients/CommonAPIClient.ts

```diff
@@ -3,31 +3,34 @@ import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
 import baseAPIClient from "./BaseAPIClient";
 
 const filterLogRecords = async (
-  building: string,
-  employeeId: string[],
-  attnTo: string[],
-  dateRange: string[],
-  tags: string[],
-  flagged: boolean,
+  building?: string,
+  employeeId?: string[],
+  attnTo?: string[],
+  dateRange?: string[],
+  tags?: string[],
+  flagged?: boolean,
+  return_all?: boolean,
 ): Promise<any> => {
   try {
     const bearerToken = `Bearer ${getLocalStorageObjProperty(
       AUTHENTICATED_USER_KEY,
       "accessToken",
     )}`;
-    const { data } = await baseAPIClient.get("/log_records", {
-      params: {
-        filters: {
-          building,
-          employeeId,
-          attnTo,
-          dateRange,
-          tags,
-          flagged,
+    const { data } = await baseAPIClient.get(
+      `/log_records?return_all=${return_all}`,
```

Comment:
`return_all` can be put as part of `params` 

`params: {filters: {...}, return_all}`